expression/builtin_other.go

```diff
@@ -361,12 +364,12 @@ func (b *builtinBitCountSig) evalInt(row types.Row) (int64, bool, error) {
 	return count, false, nil
 }
 
-// for plan cache of prepared statements
+// getParamFunctionClass for plan cache of prepared statements
 type getParamFunctionClass struct {
 	baseFunctionClass
 }
 
-//TODO: more typed functions will be added when typed parameters are supported.
+//TODO: getFunction more typed functions will be added when typed parameters are supported.
```

Comment:
Add a blank after `//`

Reply:
add blank will failed the pre-commit script by:

```
expression/builtin_other.go.372: // TODO: getFunction more typed functions will be added when typed parameters are supported. (godoc-local: TODO: -> getFunction?)
```

Reply:
```
// getFunction ....
// TODO: ....
```